README.md

```diff
@@ -101,46 +113,77 @@ Note: Omit `poetry run` in all example commands going forward if you've installe
 poetry run modelbench benchmark -m 10
 ```
 
-You should immediately see progress indicators, and depending on how loaded TogetherAI is,
-the whole run should take about 15 minutes.
+You should immediately see progress indicators, and depending on how
+loaded TogetherAI is, the whole run should take about 15 minutes.
 
 > [!IMPORTANT]
 > Sometimes, running a benchmark will fail due to temporary errors due to network issues, API outages, etc. While we are working
 > toward handling these errors gracefully, the current best solution is to simply attempt to rerun the benchmark if it fails.
 
 ### Viewing the Scores
 
-After a successful benchmark run, static HTML pages are generated that display scores on benchmarks and tests.
-These can be viewed by opening `web/index.html` in a web browser. E.g., `firefox web/index.html`. 
+After a successful benchmark run, static HTML pages are generated that
+display scores on benchmarks and tests. These can be viewed by opening
+`web/index.html` in a web browser. E.g., `firefox web/index.html`.
+
+Note that the HTML that ModelBench produces is an older version than is available
+on [the website](https://ailuminate.mlcommons.org/). Over time we'll simplify the
+direct ModelBench output to be more straightforward and more direclty useful to
+people independently running ModelBench.
+
+### Using the journal
+
+As modelbench runs, it logs each important event to the journal. That includes
```

Comment:
Should we be consistent with using ModelBench vs modelbench?

Reply:
Good catch. Here I mean the command rather than the package, so I've put it in backticks.